packages/home-spa/src/img/right-footer-cubes.svg

```diff
@@ -0,0 +1 @@
" fill="#ddd" opacity="0.8"/><path d="M125.112,104.926l39.107-22.5,39.524,21.9-39.481,23.3Z" transform="translate(-125.031 -36.792)" fill="#f3f3f3" opacity="0.803"/><path d="M126.232,51.166l36.576,21.041.223,44.547L126.185,95.429l.047-42.8Zm-1.185-.55L125,96.112l39.23,22.7-.237-47.3L127.445,50.617Z" transform="translate(-125 -27.869)" fill="#fff"/><path d="M164.248,20.37l36.965,21.354L164.054,62.675,127.445,41.71l36.8-21.326Zm0-1.355-39.182,22.7,38.983,22.3,39.562-22.274L164.248,19Z" transform="translate(-125.018 -19)" fill="#fff"/></g><g transform="translate(12088 12312.028)"><path d="M125.188,42.31l-.076,45.357,39.107-22.5.043-44.263Z" transform="translate(-125.031 -19.534)" fill="#ddd" opacity="0.8"/><path d="M218.962,87.047l-.09-44.775L179.509,20.9l-.043,44.263Z" transform="translate(-140.279 -19.534)" fill="#ddd" opacity="0.8"/><path d="M125.112,104.926l39.107-22.5,39.524,21.9-39.481,23.3Z" transform="translate(-125.031 -36.792)" fill="#f3f3f3" opacity="0.803"/><path d="M126.232,51.166l36.576,21.041.223,44.547L126.185,95.429l.047-42.8Zm-1.185-.55L125,96.112l39.23,22.7-.237-47.3L127.445,50.617Z" transform="translate(-125 -27.869)" fill="#fff"/><path d="M164.248,20.37l36.965,21.354L164.054,62.675,127.445,41.71l36.8-21.326Zm0-1.355-39.182,22.7,38.983,22.3,39.562-22.274L164.248,19Z" transform="translate(-125.018 -19)" fill="#fff"/></g><g transform="translate(11984 12313.028)"><path d="M125.188,42.31l-.076,45.357,39.107-22.5.043-44.263Z" transform="translate(-125.031 -19.534)" fill="#ddd" opacity="0.8"/><path d="M218.962,87.047l-.09-44.775L179.509,20.9l-.043,44.263Z" transform="translate(-140.279 -19.534)" fill="#ddd" opacity="0.8"/><path d="M125.112,104.926l39.107-22.5,39.524,21.9-39.481,23.3Z" transform="translate(-125.031 -36.792)" fill="#f3f3f3" opacity="0.803"/><path d="M126.232,51.166l36.576,21.041.223,44.547L126.185,95.429l.047-42.8Zm-1.185-.55L125,96.112l39.23,22.7-.237-47.3L127.445,50.617Z" transform="translate(-125 -27.869)" fill="#fff"/><path d="M164.248,20.37l36.965,21.354L164.054,62.675,127.445,41.71l36.8-21.326Zm0-1.355-39.182,22.7,38.983,22.3,39.562-22.274L164.248,19Z" transform="translate(-125.018 -19)" fill="#fff"/></g><g transform="translate(12036 12358.028)"><path d="M125.188,42.31l-.076,45.357,39.107-22.5.043-44.263Z" transform="translate(-125.031 -19.534)" fill="#ddd" opacity="0.8"/><path d="M218.962,87.047l-.09-44.775L179.509,20.9l-.043,44.263Z" transform="translate(-140.279 -19.534)" fill="#ddd" opacity="0.8"/><path d="M125.112,104.926l39.107-22.5,39.524,21.9-39.481,23.3Z" transform="translate(-125.031 -36.792)" fill="#f3f3f3" opacity="0.803"/><path d="M126.232,51.166l36.576,21.041.223,44.547L126.185,95.429l.047-42.8Zm-1.185-.55L125,96.112l39.23,22.7-.237-47.3L127.445,50.617Z" transform="translate(-125 -27.869)" fill="#fff"/><path d="M164.248,20.37l36.965,21.354L164.054,62.675,127.445,41.71l36.8-21.326Zm0-1.355-39.182,22.7,38.983,22.3,39.562-22.274L164.248,19Z" transform="translate(-125.018 -19)" fill="#fff"/></g></g></svg>
```

Comment:
Can we move these SVGs to `/assets`?

Or is very specific to Home SPA?

Reply:
@deshmukhmayur It is currently only used in Home-SPA, in future we may want to use it in the footer component.
All of the assets regardless if they are specific or not to Home-SPA should be moved to `/assets` IMO. But for now we can keep them here because there aren't other duplications of the same asset